test/setup/main.tf

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "project" {
+  source  = "terraform-google-modules/project-factory/google"
+  version = "~> 3.0"
+
+  name              = "ci-{{ cookiecutter.module_name|replace('-', '_') }}"
```

Comment:
Please add a valid name, for example, "ci-log-export"

Reply:
@ingwarr, thank you for review! [Fixed](https://github.com/terraform-google-modules/terraform-google-log-export/compare/07b614e0d09443a93a5e1eae9dbc3f053fd832c9..2976460f55f997027cbd58e300dfe5d00e427140)